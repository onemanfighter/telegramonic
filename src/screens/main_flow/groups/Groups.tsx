const Groups = () => {
  return (
    <div>
      <h1>Groups</h1>
    </div>
  );
};

export default Groups;
