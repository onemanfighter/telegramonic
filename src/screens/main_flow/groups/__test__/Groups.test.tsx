import { render } from '@testing-library/react';
import Groups from '../Groups';

describe('Groups screen', () => {
  it('should render correctly', () => {
    const { container } = render(<Groups />);

    expect(container).toMatchSnapshot();
  });
});
