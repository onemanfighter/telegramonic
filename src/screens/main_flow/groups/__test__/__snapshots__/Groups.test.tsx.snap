// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Groups screen should render correctly 1`] = `
<div>
  <div>
    <h1>
      Groups
    </h1>
  </div>
</div>
`;
